packages/flutter/lib/src/material/app.dart

```diff
@@ -29,7 +29,7 @@ import 'page.dart';
 import 'scaffold.dart' show ScaffoldMessenger, ScaffoldMessengerState;
 import 'scrollbar.dart';
 import 'theme.dart';
-import 'theme_data.dart';
+import 'theme_data.dart' show MaterialTapTargetSize;
```

Comment:
Although `theme.dart` exports `ThemeData`, it doesn't export the `MaterialTapTargetSize` (which is the reason for the added import to `theme_data.dart`

Reply:
Ah, I see! Maybe it should then. I looked around and noticed it is used multiple times like this. Can you change `theme.dart` to also export MaterialTapTargetSize as well instead?